kernel_gateway/services/kernels/manager.py

```diff
@@ -101,7 +101,7 @@ def start_kernel(self, *args, **kwargs):
                     # executed
                     if self.parent.personality.should_seed_cell(code):
                         client.execute(code)
-                        msg = client.shell_channel.get_msg(block=True)
+                        msg = client.get_shell_msg(block=True)
```

Comment:
I think the `block=True` needs to be removed.

@davidbrochart - Should we update the [migration guide](https://jupyter-client.readthedocs.io/en/latest/migration.html#zmqsocketchannel) to indicate what calls using `block=True` should do since we only address the `block=False` case wrt to a `timeout` value?  I found removing the `block=True` to be successful but am not entirely sure.

Reply:
`block=True` was the default value, so it could have been omitted previously. It translates to a value of `timeout>0` now, which is the default value, so there's nothing to do, and `block=True` just has to be removed. We can clarify this in the migration guide, and also state that using e.g. `client.get_shell_msg()` should be used instead of `client.shell_channel.get_msg()`. I opened https://github.com/jupyter/jupyter_client/pull/685.